react/hooks/useFetchMore.js

```diff
@@ -11,6 +11,39 @@ export const FETCH_TYPE = {
   PREVIOUS: 'previous',
 }
 
+function reducer(state, action) {
+  const { maxItemsPerPage, to, from } = action.args
+  switch (action.type) {
+    case 'RESET':
+      return {
+        page: 1,
+        nextPage: 2,
+        previousPage: 0,
+        from: 0,
+        to: maxItemsPerPage - 1,
+      }
+    case 'NEXT_PAGE':
+      return {
+        ...state,
+        page: state.nextPage,
+        nextPage: state.nextPage + 1,
+        to,
+      }
+    case 'PREVIOUS_PAGE':
+      return {
+        ...state,
+        page: state.previousPage,
+        previousPage: state.previousPage - 1,
+        from,
+      }
+
+    case 'ROLLBACK':
```

Comment:
this looks wrong, it is not changing anything, you should pass the variables to rollback to in here